### PR TITLE
review: remove files for review

### PR DESCRIPTION
**Pull request by ro-ka**
Creating a PR to do a review on the files.  😼

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,48 +0,0 @@
-{
-  "name": "@caerostris/parallel-transform-stream",
```
Comment (ro-ka) [inline package.json:2]: Does it make sense to publish with a prefix? There’s no module with that name and it’s developed @ubilabs.

File: package.json [hunk 2]
```diff
@@ -1,48 +0,0 @@
...
-  "version": "0.0.1",
-  "description": "Transform streams in parallel",
-  "main": "dist/parallel-transform.js",
-  "scripts": {
-    "build": "babel ./src --out-dir ./dist",
```
Comment (ro-ka) [inline package.json:7]: So it should work with older Node versions too, right?

File: .release.json [hunk 3]
```diff
@@ -1,36 +0,0 @@
-{
-  "non-interactive": false,
-  "dry-run": true,
```
Comment (ro-ka) [inline .release.json:3]: No real release?

File: .release.json [hunk 4]
```diff
@@ -1,36 +0,0 @@
...
-    "pushRepo": null,
-    "beforeStartCommand": false,
-    "beforeStageCommand": false,
-    "afterReleaseCommand": false
-  },
-  "dist": {
-    "baseDir": "dist"
-  },
-  "npm": {
-    "publish": false,
```
Comment (ro-ka) [inline .release.json:25]: Why not publish on release?

File: README.md [hunk 5]
```diff
@@ -1,125 +0,0 @@
-# Parallel Transform Stream
-
-A NodeJS transform stream which runs transformations in parallel and preserves input order.
-
-```sh
-npm install parallel-transform-stream --save
```
Comment (ro-ka) [inline README.md:6]: Either add a prefix here, too, or remove it in general.

File: package.json (hunk 1 above)
Comment (Caerostris) [inline package.json]: Nope, I was going to remove the prefix before release :)

File: README.md [hunk 6]
```diff
@@ -1,125 +0,0 @@
...
-## Documentation
-
-All classes extending the `ParallelTransform` class must implement the method `_parallelTransform`.  
-They may implement `_parallelFlush`, although this is not required.
-
-### API
-
-**ParallelTransform.create(transform, flush = function(done) { done(); }, defaultOptions = {})**
-
-* `transform` `<Function>` The \_transform function of the stream. See below for more details
```
Comment (ro-ka) [inline README.md:54]: Link the _below_ to the section mentioned.

File: README.md [hunk 7]
```diff
@@ -1,125 +0,0 @@
...
-* `flush` `<Function>` The \_flush function of the stream. See below for more details
```
Comment (ro-ka) [inline README.md:55]: Same, link the _below_.

File: .release.json (hunk 4 above)
Comment (Caerostris) [inline .release.json]: Wanted to make sure I don't accidentally publish anything before the review 👍

File: README.md [hunk 8]
```diff
@@ -1,125 +0,0 @@
...
-
-All classes extending `ParallelTransform` must implement the `_parallelTransform` method, and may implement the `_parallelFlush` method.
-
-**ParallelTransform._parallelTransform(chunk, encoding, callback)**
-
-* `chunk` `<Buffer>` | `<String>` The chunk to be transformed.
-* `encoding` `<String>` If the chunk is a string, then this is the encoding type. If chunk is a buffer, then this is the special value - 'buffer', ignore it in this case.
-* `callback` `<Function>` A callback function to be called after the supplied chunk has been processed. The first argument passed to the callback must be an error which has occurred during transformation, or `null`. The second argument is the result. The stream will stop processing transforms and emit an `error` event instantly if the error passed to the callback function was not `null`.
-
-Please note that, as opposed to traditional NodeJS transform streams, you **MUST NOT** call `this.push` directly. Emit values through the callback function instead.
```
Comment (ro-ka) [inline README.md:70]: Highlight the _Please note_.

File: package.json (hunk 2 above)
Comment (Caerostris) [inline package.json:7]: Test cases do not pass on Node < v4. It won't run on 0.12, unless my test setup is broken. It certainly won't run on v0.10.

File: src/parallel-transform.js [hunk 9]
```diff
@@ -1,194 +0,0 @@
...
-import stream from 'stream';
-import cyclist from 'cyclist';
-
-const _maxParallel = new WeakMap(),
-  _destroyed = new WeakMap(),
-  _flushed = new WeakMap(),
-  _buffer = new WeakMap(),
-  _top = new WeakMap(),
-  _bottom = new WeakMap(),
-  _ondrain = new WeakMap();
```
Comment (ro-ka) [inline src/parallel-transform.js:11]: Should be `_onDrain` or is it used in lowercase somewhere else?

File: src/parallel-transform.js [hunk 10]
```diff
@@ -1,194 +0,0 @@
...
-    }
-
-    _destroyed.set(this, true);
-    this.emit('close');
-  }
-
-  /**
-   * Parallises calls to this._transformFunction
-   * @param {?}        chunk The chunk of data to be transformed
-   * @param {string}   encoding Encoding, if it `chunk` is a string
```
Comment (ro-ka) [inline src/parallel-transform.js:78]: Typo.

File: src/parallel-transform.js [hunk 11]
```diff
@@ -1,194 +0,0 @@
...
-      // abort on error
-      if (err) {
-        this.emit('error', err);
-        this.push(null);
-        this.destroy();
-        return;
-      }
-
-      // insert result into corresponding place in buffer
-      const result = typeof data === 'undefined' || data === null ? null : data;
```
Comment (ro-ka) [inline src/parallel-transform.js:99]: The second comparison is not needed. If `data === null`, it looks like this:

``` js
typeof null === 'undefined' || null === null ? null : null;
```

File: README.md (hunk 5 above)
Comment (Caerostris) [inline README.md:6]: A prefix? You mean, as in `node-parallel-transform-stream`? I think having a `node-` prefix in the github repo makes sense, as not all repos in [github.com/ubilabs](github.com/ubilabs) are node modules. Having a `node-` prefix as package name for the **Node** Package Manager feels a bit redundant...
(edit: fix typo)

File: src/parallel-transform.js [hunk 12]
```diff
@@ -1,194 +0,0 @@
...
-    _top.set(this, pos + 1);
-
-    this._parallelTransform(chunk, encoding, (err, data) => {
-      if (_destroyed.get(this)) {
-        return;
-      }
-
-      // abort on error
-      if (err) {
-        this.emit('error', err);
```
Comment (ro-ka) [inline src/parallel-transform.js]: Isn’t the convention that the error is the first var? See how you use the `_paralllelTransform`.

File: src/parallel-transform.js (hunk 11 above)
Comment (Caerostris) [inline src/parallel-transform.js]: Good point 👍

File: src/parallel-transform.js (hunk 12 above)
Comment (Caerostris) [inline src/parallel-transform.js]: Which line?

File: test/parallel-transform.js [hunk 13]
```diff
@@ -1,149 +0,0 @@
...
-    t.true(transformStub.calledOnce);
-    t.end();
-  });
-
-  transformInstance.write(data);
-  transformInstance.end();
-});
-
-test('should allow instantiation via function call', t => {
-  const PStream = ParallelTransform.create({
```
Comment (ro-ka) [inline test/parallel-transform.js:76]: Name it ParallelStream. No hard to guess naming.

**Comment by ro-ka** [comment]
Add a travis.yml.

**Comment by ro-ka** [comment]
🏁

File: test/parallel-transform.js (hunk 13 above)
Comment (Caerostris) [inline test/parallel-transform.js:76]: 👍

File: package.json (hunk 2 above)
Comment (ro-ka) [inline package.json:7]: Okay, just thought as it is ES5 it’ll work. But there were breaking changes. Fine then.

File: README.md (hunk 5 above)
Comment (ro-ka) [inline README.md:6]: No no, the `@caerostris/` prefix.  ;)

File: src/parallel-transform.js (hunk 12 above)
Comment (ro-ka) [inline src/parallel-transform.js]: 85, directly above.

File: src/parallel-transform.js (hunk 9 above)
Comment (Caerostris) [inline src/parallel-transform.js]: Yup, should be camel case.

**Comment by ro-ka** [comment]
So this module is ready for a `1.0.0` release?

File: README.md (hunk 5 above)
Comment (Caerostris) [inline README.md:6]: Ahh haha I see! That one was just for testing anyways :)

**Pull request closed without merging**